src/pages/vrf/v2/examples/lootbox.mdx

```diff
@@ -0,0 +1,410 @@
+  }
+---
+
+import { Aside, CodeSample } from "@components"
+import { Tabs } from "@components/Tabs"
+
+The lootbox utility contract uses Chainlink VRF to determine a random reward that you can distribute as a [Loot Box](https://en.wikipedia.org/wiki/Loot_box) to users. This allows you to issue rewards fairly and transparently using verified randomness that the recipients can verify on-chain.
+
+{/* prettier-ignore */}
+<div class="remix-callout">
+  <a href="https://github.com/smartcontractkit/quickstarts-lootbox">See the code on GitHub</a>
+</div>
+
+## Overview
+
+If you need to create a reward distribution dApp, this example demonstrates a basic example to help you get started. In this tutorial, you will use example code from the [smartcontractkit/quickstarts-lootbox](https://github.com/smartcontractkit/quickstarts-lootbox) repository. You will configure the example to work on a testnet of your choice, use an RPC endpoint of your choice, and issue tokens to an end-user in a lootbox contract. This example uses the [HardHat](https://hardhat.org/) framework to help you complete the testing and deployment steps.
+
+This example contract supports [ERC-20](https://eips.ethereum.org/EIPS/eip-20), [ERC-721](https://eips.ethereum.org/EIPS/eip-721), and [ERC-1155](https://eips.ethereum.org/EIPS/eip-1155) tokens as rewards. The rewards are distributed from a pool of tokens that are transferred to the lootbox contract when you deploy it.
+
+<Aside type="caution" title="Disclaimer">
+  This tutorial is an example to help you learn how to integrate Chainlink services and integrate them into your own
+  dApps. This template is provided "AS IS" without warranties of any kind, has not been audited, and may be missing key
+  checks or error handling to make the usage of the product more clear. Do not use the code in this example in a
+  production environment without completing your own audits and application of best practices.
+</Aside>
+
+## Before you begin
+
+Before you start this tutorial, complete the following items:
+
+- Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+  - Run `git --version` to check the installation. You should see an output similar to `git version x.x.x`.
+- Install [Nodejs](https://nodejs.org/en/) 16.0.0 or higher
+  - Run `node --version` to check the installation. You should see an output similar to `v16.x.x`.
+  - Optionally, you can also [Install Yarn](https://classic.yarnpkg.com/lang/en/docs/install/) and use it to run this example instead of using `npm`.
+- [Create an Etherscan API key](https://docs.etherscan.io/getting-started/viewing-api-usage-statistics) if you do not already have one.
+- Create an account with [Infura](https://www.infura.io/) or [Alchemy](https://www.alchemy.com/) to use as an RPC endpoint if you do not already have one. Alternatively, you can [use your own testnet clients](/chainlink-nodes/resources/run-an-ethereum-client).
```

Comment:
```suggestion
- If you are new to smart contract development, learn how to [Deploy Your First Smart Contract](/getting-started/deploy-your-first-contract).
- Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
  - Run `git --version` to check the installation. You should see an output similar to `git version x.x.x`.
- Install [Nodejs](https://nodejs.org/en/) 16.0.0 or higher
  - Run `node --version` to check the installation. You should see an output similar to `v16.x.x`.
  - Optionally, you can also [Install Yarn](https://classic.yarnpkg.com/lang/en/docs/install/) and use it to run this example instead of using `npm`.
- [Create an Etherscan API key](https://docs.etherscan.io/getting-started/viewing-api-usage-statistics) if you do not already have one.
- Create an account with [Infura](https://www.infura.io/) or [Alchemy](https://www.alchemy.com/) to use as an RPC endpoint if you do not already have one. Alternatively, you can [use your own testnet clients](/chainlink-nodes/resources/run-an-ethereum-client).
```